maps/misc/misc_map_items.dm

```diff
@@ -0,0 +1,41 @@
+/obj/machinery/mob_printer/deathmatch
+	name = "deathmatch spawner"
+	desc = "Will summon bloodthirsty murderers at random"
+	cooldown_duration = 10 SECONDS
+	print_path = /mob/living/carbon/human
+	var/obj/abstract/loadout/to_load = /obj/abstract/loadout/deathmatch
+
+/obj/machinery/mob_printer/deathmatch/finalize_spawn(var/mob/living/M)
+	new to_load(loc, M)
+	return M
+
+/obj/abstract/loadout/deathmatch
```

Comment:
loadout datums hurt my soul

Reply:
outfit datums are still in the cuck branch

Reply:
i know
i'll finish them eventually :tm: